src/surforama/io/mesh.py

```diff
@@ -23,3 +26,53 @@ def read_obj_file(file_path):
     values = np.ones((len(verts),))
 
     return verts, faces, values
+
+
+def convert_mask_to_mesh(
+    mask: np.ndarray,
+    barycentric_area: float = 1.0,
+    smoothing: int = 10,
+):
+    """
+    Convert a binary mask to a mesh.
+
+    Parameters
+    ----------
+    mask : np.ndarray
+        A binary mask.
+    barycentric_area : float, optional
+        The area of each triangle in the mesh, by default 1.0
```

Comment:
Is this the target area (i.e., it doesn't actually guarantee that this area is uniformly achieved)? If so, I think we should specify

```suggestion
        The target area of each triangle in the mesh, by default 1.0
```

Reply:
You're right, it's only the target area.
But I checked again and it's actually the barycentric area of each _vertex_, not each _triangle_.
I.e. the clustering will results in the specified number of vertices.

I also updated the docstring for this